sound/hybrid_font.h

```diff
@@ -527,13 +527,14 @@ class HybridFont : public SaberBase, public Looper {
 	if (num_alternatives) {
 	  if (SaberBase::sound_number == -1) {
 	    // Next alternative
-	    if (++current_alternative >= num_alternatives)  current_alternative = 0;
+	    ++current_alternative;
 	  } else {
 	    // Select a specific alternative.
 	    current_alternative = std::min<int>(SaberBase::sound_number, num_alternatives);
```

Comment:
If you don't want wrap-around, then the correct fix is to change this to num_alternatives - 1
(I think, I don't remember how I lined this variables up actually...)